fuji_server/helper/metadata_mapper.py

```diff
@@ -60,6 +60,7 @@ def flip_dict(dict_to_flip):
         "right_holder": {"label": "License", "sameAs": "http://purl.org/dc/terms/rightsHolder"},
         "object_size": {"label": "Object Size", "sameAs": "http://purl.org/dc/terms/extent"},
         "language": {"label": "Language", "sameAs": "http://purl.org/dc/terms/language"},
+        "license_path": {"label": "License Path", "sameAs": None},
```

Comment:
What is the difference between license and license_path, can this be merged?

Reply:
`license` contains the license name/identifier, so something like `mit` - we use this the same for the GitHub harvest as it's used by the metadata harvester. `license_path` on the other hand is the _file path_ to the license file in the repository, e.g. `./LICENSE.TXT`. It's relevant for one of the CESSDA-specific metric tests. 

I'm not sure how they might be merged since we need both pieces of information for the tests. Happy to adjust this if you have any suggestions though!

Reply:
Ah I see, this is indeed different, so let's keep it